make-webpack-config.js

```diff
@@ -72,7 +72,9 @@ module.exports = function(rules, options) {
       new UglifyJsPlugin({
         uglifyOptions: {
           mangle: specialOptions.mangle,
-          compress: specialOptions.mangle,
+          compress: specialOptions.mangle ? {
+            dead_code: true
```

Comment:
This option ensures that the development-only condition is completely removed from production builds, along with the modules needed in that block:

https://github.com/swagger-api/swagger-ui/blob/8480c13b8e2f1425e048e6f1cf30e837d7c35ab8/src/core/index.js#L10-L12